pkg/operator/controller/configauditreport.go

```diff
@@ -409,7 +409,12 @@ func (r *ConfigAuditReportReconciler) processCompleteScanJob(ctx context.Context
 
 	logsStream, err := r.LogsReader.GetLogsByJobAndContainerName(ctx, job, r.Plugin.GetContainerName())
 	if err != nil {
-		return fmt.Errorf("getting logs: %w", err)
+		if errors.IsNotFound(err) {
+			log.V(1).Info("Ignoring not found job", "job", fmt.Sprintf("%s/%s", job.Namespace, job.Name), "owner", owner)
+			log.V(1).Info("Deleting complete scan job", "owner", owner)
+			return r.deleteJob(ctx, job)
```

Comment:
@mycodeself Probably `r.deleteJob` will not find a job to delete, due to job `not found`.

Maybe is better just to `return ctrl.Result{}, nil` to remove it from the cache :
https://github.com/kubernetes-sigs/controller-runtime/blob/master/pkg/internal/controller/controller.go#L332

Reply:
@chen-keinan but `return ctrl.Result{}, nil` doesn't fit the signature of `processCompleteScanJob` it just returns an error. I guess that I should just `return nil` right?

Reply:
@mycodeself FYI, The complete solution has been implemented by #956

Reply:
Oh ok @chen-keinan thanks for letting me know! I'll watch this PR to continue learning!